### ISSUE TITLE
fix: CSS class ordering in component rendering

### ISSUE THREAD

**Issue by tlmii** [microsoft/fluentui-blazor#741]
# 🐛 Bug Report

When components have a Class attribute for specifying additional classes AND they have their own default class used as part of their styling, the custom class is listed first, and the default class is listed second. This means that it is not possible to override styles in the default class via another class - it must be done via style.

## 💻 Repro or Code Sample

```razor
<FluentBodyContent Class="custom-body-content">
    @Body
</FluentBodyContent>
```

```css
.custom-body-content {
    margin-right: 10px;
}
```

## 🤔 Expected Behavior

```html
<div class="body-content custom-body-content">
</div>
```

## 😯 Current Behavior

```html
<div class="custom-body-content body-content">
</div>
```

## 💁 Possible Solution

It looks like the ordering would just need to be different in places where the CssBuilder is used, like [this](https://github.com/microsoft/fluentui-blazor/blob/f6ef9cd6398ad7bbf5a526e15a5669dd5b2b1617/src/Core/Components/BodyContent/FluentBodyContent.razor.cs#L9):

Current:
```csharp
protected string? ClassValue => new CssBuilder(Class)
    .AddClass("body-content")
    .Build();
```

Changed:
```csharp
protected string? ClassValue => new CssBuilder("body-content")
    .AddClass(Class)
    .Build();
```

## 🔦 Context

The current design makes it hard to avoid using inline styles.

A [quick code search](https://github.com/search?q=repo%3Amicrosoft%2Ffluentui-blazor%20ClassValue%20%3D%3E%20new%20CssBuilder&type=code) shows this impacts several components, but definitely not all of them (since many don't have a default style).

**Comment by dvoituron** [comment]
You're right. We'll change that as soon as possible.

**Comment by dvoituron** [comment]
That will be included in the next release, via #745

**Comment by tlmii** [comment]
Thanks for the quick response!
